code-of-conduct.md

```diff
@@ -7,6 +7,9 @@ As members of the community,
 - We pledge to welcome those interested in joining the community, and realize that including people with a variety of opinions and backgrounds will only serve to enrich our community. In particular, discussions relating to pros/cons of various technologies, programming languages, and so on are welcome, but these should be done with respect, taking proactive measure to ensure that all participants are heard and feel confident that they can freely express their opinions.
 - We pledge to welcome questions and answer them respectfully, paying particular attention to those new to the community. We pledge to provide respectful criticisms and feedback in forums, especially in discussion threads resulting from code contributions.
 - We pledge to be conscientious of the perceptions of the wider community and to respond to criticism respectfully. We will strive to model behaviors that encourage productive debate and disagreement, both within our community and where we are criticized. We will treat those outside our community with the same respect as people within our community.
+
+*If any community member ever feels like they can not respond to a public forum or private digital communication regarding the Astropy Project or Python in a supportive and helpful way, do not respond. Please forward the communication to a [Community Coordinator](http://www.astropy.org/team.html), the Coordination Committee (coordinators@astropy.org) or confidential@astropy.org.*
```

Comment:
Hmm, I don't like the way this renders.  It breaks up the "We pledge" flow below.  I wonder if it might be better if it actually be put directly into the web page copy of the CoC as a footnote or similar sort of form that doesn't break up the flow of the primary bullet points?

Reply:
(I guess the solution to this depends somewhat on my comment further up about whether this is the "true" version of the CoC or if this is just a draft copy for us to integrate into the web site as the "true" version.)

Reply:
Yes I was actually thinking about this again and I think this is more of a bit about how to act on the CC than a new bullet point. It could be mentioned around the same place where we mention who to contact

Reply:
Can we just pick one point of contact for sending the communication instead of three choices?  I would just say to send to the coordination committee (coordinators@astropy.org).  Where does confidential@astropy.org go to?

Reply:
It's important to give people options for who to contact, especially since our community is so small. For example, what if the problem is with someone on the coordinating committee?

Confidential@astropy.org will go to the Ombudsperson (that should be added!).

Reply:
I modified it and moved the language to the end so as to not break up the flow. I also added explicitly that confidential goes to the Ombudsperson.